### ISSUE TITLE
[ADAP-959] [Bug] macro describe_materialized_view does not work with redshift serverless

### ISSUE THREAD

**Issue by jmeiring-sc** [dbt-labs/dbt-redshift#641]
### Is this a new bug in dbt-redshift?

- [X] I believe this is a new bug in dbt-redshift
- [X] I have searched the existing issues, and I could not find an existing issue for this bug

### Current Behavior

When doing a `dbt run` on a redshift serverless cluster, permission denied errors will happen when dbt attempts to query the svt_mv_info table. This is due to that table not being queryable by any user in redshift serverless [as described here](https://docs.aws.amazon.com/redshift/latest/mgmt/serverless-monitoring.html#serverless_views-monitoring).

This can be remedied by a custom macro for `redshift__describe_materialized_view`, which references the `svv_mv_info` view instead of the `svt_mv_info` table as shown below

```
{% macro redshift__describe_materialized_view(relation) %}
    {#-
        These need to be separate queries because redshift will not let you run queries
        against svv_table_info and pg_views in the same query. The same is true of svv_redshift_columns.
    -#}

    {%- set _materialized_view_sql -%}
        select
            tb.database,
            tb.schema,
            tb.table,
            tb.diststyle,
            tb.sortkey1,
            mv.autorefresh
        from svv_table_info tb
        left join svv_mv_info mv
            on mv.database_name = tb.database
            and mv.schema_name = tb.schema
            and mv.name = tb.table
        where tb.table ilike '{{ relation.identifier }}'
        and tb.schema ilike '{{ relation.schema }}'
        and tb.database ilike '{{ relation.database }}'
    {%- endset %}
    {% set _materialized_view = run_query(_materialized_view_sql) %}

    {%- set _query_sql -%}
        select
            vw.definition
        from pg_views vw
        where vw.viewname = '{{ relation.identifier }}'
        and vw.schemaname = '{{ relation.schema }}'
        and vw.definition ilike '%create materialized view%'
    {%- endset %}
    {% set _query = run_query(_query_sql) %}

    {% do return({'materialized_view': _materialized_view, 'query': _query}) %}

{% endmacro %}
```

### Expected Behavior

properly introspect the materialized view in redshift serverless

### Steps To Reproduce

`dbt run` with a model with materialization=materialized_view on a redshift serverless cluster

### Environment

```markdown
- OS:
- Python: 
- dbt-core: 1.6.2
- dbt-redshift:1.6.2
```

**Comment by dbeatty10** [comment]
Thanks for reporting this and including a solution @jmeiring-sc ! 🤩

The update you are suggesting [here](https://github.com/dbt-labs/dbt-redshift/blob/fe30bb1ca4989de77fb77d05839f019a65d3f6fc/dbt/include/redshift/macros/relations/materialized_view/describe.sql#L16-L18) looks really simple:
```diff
<         left join stv_mv_info mv
<             on mv.db_name = tb.database
<             and mv.schema = tb.schema
---
>         left join svv_mv_info mv
>             on mv.database_name = tb.database
>             and mv.schema_name = tb.schema
```

Would you be interested in raising a PR, by any chance? No worries if not.

**Comment by jmeiring-sc** [comment]
sure, I'll put in a PR today

**Comment by dbeatty10** [comment]
Awesome @jmeiring-sc !

When you raise that PR, could you add `resolves #641` to the top of the PR and tag `@dbeatty10` with a comment? That way I can make sure our engineering team knows it's ready for review.

**Comment by mikealfare** [comment]
This has not been backported nor released yet. I'm re-opening and will close it when we confirm that it's released.

**Comment by mikealfare** [comment]
@jmeiring-sc This should be available in `dbt-redshift~=1.7.1`. But I noticed that you reported it in `dbt-redshift==1.6.2`. We'll get the backport merged and it should go out in the next release next Thursday.
